README.md

```diff
@@ -0,0 +1,422 @@
+| [BookStoreApplicationTests.java](https://github.com/oksana-miazina/mate_book_store/blob/master/src/test/java/mate/academy/bookstore/BookStoreApplicationTests.java) | some description `src/test/java/mate/academy/bookstore/BookStoreApplicationTests.java` |
+
+</details>
+
+---
+
+##  Getting Started
+
+***Requirements***
+
+Ensure you have the following dependencies installed on your system:
+
+* **Java**: `version 17`
+
+###  Installation
+
+1. Clone the mate_book_store repository:
+
+```sh
+git clone https://github.com/oksana-miazina/mate_book_store
+```
+
+2. Change to the project directory:
+
+```sh
+cd mate_book_store
+```
+
+3. Install the dependencies:
+
+```sh
+mvn clean install
+```
+
+###  Running mate_book_store
+
+Use the following command to run mate_book_store:
+
+```sh
+java -jar target/myapp.jar
+```
+
+###  Tests
+
+To execute tests, run:
+
+```sh
+mvn test
+```
+
+---
+
+##  Project Roadmap
+
+- [X] `► INSERT-TASK-1`
+- [ ] `► INSERT-TASK-2`
+- [ ] `► ...`
+
+---
+
+##  Contributing
+
+Contributions are welcome! Here are several ways you can contribute:
+
+- **[Submit Pull Requests](https://github.com/oksana-miazina/mate_book_store/blob/main/CONTRIBUTING.md)**: Review open PRs, and submit your own PRs.
+- **[Join the Discussions](https://github.com/oksana-miazina/mate_book_store/discussions)**: Share your insights, provide feedback, or ask questions.
+- **[Report Issues](https://github.com/oksana-miazina/mate_book_store/issues)**: Submit bugs found or log feature requests for Mate_book_store.
+
+<details closed>
+    <summary>Contributing Guidelines</summary>
+
+1. **Fork the Repository**: Start by forking the project repository to your GitHub account.
+2. **Clone Locally**: Clone the forked repository to your local machine using a Git client.
+   ```sh
+   git clone https://github.com/oksana-miazina/mate_book_store
+   ```
+3. **Create a New Branch**: Always work on a new branch, giving it a descriptive name.
+   ```sh
+   git checkout -b new-feature-x
+   ```
+4. **Make Your Changes**: Develop and test your changes locally.
+5. **Commit Your Changes**: Commit with a clear message describing your updates.
+   ```sh
+   git commit -m 'Implemented new feature x.'
+   ```
+6. **Push to GitHub**: Push the changes to your forked repository.
+   ```sh
+   git push origin new-feature-x
+   ```
+7. **Submit a Pull Request**: Create a PR against the original project repository. Clearly describe the changes and their motivations.
+
+Once your PR is reviewed and approved, it will be merged into the main branch.
+
+</details>
+
+---
+
+##  License
+
+This project is protected under the [The Unlicense](https://choosealicense.com/licenses/unlicense/) License. For more details, refer to the [LICENSE](https://choosealicense.com/licenses/) file.
```

Comment:
From header I see that there is not licence (not specified), but here I see that there is one